### PR TITLE
Add name property to horizontal thumb in Syntax Visualizer for accessibility

### PR DESCRIPTION
**Pull request by chborl**
The name property is empty for the horizontal thumb, which is on the bottom of the syntax tree item page of the Syntax visualizer pane.  This PR adds the name for the horizontal thumb so it can be displayed and read by the screen reader.

### CODE REVIEW

**Comment by chborl** [comment]
@dotnet/roslyn-ide

**Comment by jinujoseph** [comment]
@chborl can you map the bug number for reference pls

**Comment by chborl** [comment]
Fixes this bug: https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=483432

File: src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Control/SyntaxVisualizerControl.xaml [hunk 1]
```diff
@@ -87,7 +87,7 @@
                 </ContextMenu>
             </TreeView.ContextMenu>
         </TreeView>
-        <GridSplitter Grid.Row="2" ResizeDirection="Rows" HorizontalAlignment="Stretch" Height="5"/>
+        <GridSplitter Grid.Row="2" AutomationProperties.Name ="Horizontal" ResizeDirection="Rows" HorizontalAlignment="Stretch" Height="5"/>
```
Comment (jasonmalinowski) [inline src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Control/SyntaxVisualizerControl.xaml]: No space before the =.

**Pull request merged**